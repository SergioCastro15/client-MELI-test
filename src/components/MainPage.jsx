import React from 'react'

export const MainPage = () => {
    return (
        <div>
            <h1>Bienvenido por favor realiza una busqueda</h1>
        </div>
    )
}
