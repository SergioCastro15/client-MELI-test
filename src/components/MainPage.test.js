import React from 'react'
import { render } from '@testing-library/react'
import { MainPage } from './MainPage'


it('should take a snapshot', () => {
    const { asFragment } = render(<MainPage />)
    expect(asFragment(<MainPage />)).toMatchSnapshot()
})