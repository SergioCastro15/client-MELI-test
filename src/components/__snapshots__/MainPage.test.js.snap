// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Bienvenido por favor realiza una busqueda
    </h1>
  </div>
</DocumentFragment>
`;
